octobot_trading/exchanges/rest_exchange.py

```diff
@@ -38,6 +38,8 @@ class RestExchange(AbstractExchange):
     CCXT library wrapper
     """
 
+    CCXT_CLIENT_LOGIN_OPTIONS = {}
```

Comment:
What is it used for ? Should we put it in config.json ?

Reply:
It's overwritten by exchange client that need login options like https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/137/files#diff-f9edc36c5fc59e1e37cf878054baf95bR40